package.json

```diff
@@ -2,7 +2,8 @@
   "name": "react-waypoint",
   "version": "7.2.0",
   "description": "A React component to execute a function whenever you scroll to an element.",
-  "main": "build/waypoint.js",
+  "main": "build/index.cjs.js",
+  "module": "build/index.esm.js",
```

Comment:
If you want to be compatible with what node will support, you'll want to do `build/index.js` and `build/index.mjs` here.